README.md

```diff
@@ -13,48 +13,43 @@ Also uses:
 * [Trade Tariff Suite](https://github.com/alphagov/trade-tariff-suite)
 * [Trade Tariff Oracle](https://github.com/alphagov/trade-tariff-oracle)
 
-## Setup
+## Development
 
-### Dependencies (OS X using Homebrew)
+### Dependencies
 
-1. ElasticSearch & MySQL & Redis
+  - ElasticSearch
+  - Postgresql
+  - Redis
+  - Ruby 2.2.3
+  - Foreman gem
```

Comment:
yeah we could improve this setup checking the service and install it if is not installed install it if it's linux or mac.. but we should discuss it first..maybe some people will not want to install but use something like docker or don't want to use a  homebrew package..